RELEASING/README.md

```diff
@@ -77,23 +77,23 @@ Usage (ZSH):
 
 Example:
 ```bash
-    source set_release_env.sh 0.37.0rc1 myid@apache.org
+    source set_release_env.sh 0.38.0rc1 myid@apache.org
 ```
 
 The script will output the exported variables. Here's example for 0.37.0rc1:
 
 ```
     Set Release env variables
-    SUPERSET_VERSION=0.37.0
+    SUPERSET_VERSION=0.38.0
```

Comment:
The text on line 83 should probably also be updated (it's referencing 0.38.0rc1).